sound/soc/intel/boards/Kconfig

```diff
@@ -599,6 +599,24 @@ config SND_SOC_INTEL_SOF_DA7219_MAX98373_MACH
 
 endif ## SND_SOC_SOF_JASPERLAKE
 
+if SND_SOC_SOF_TIGERLAKE
+
+config SND_SOC_INTEL_TGL_RT1308_MACH
+	tristate "TGL with RT1308 in I2S Mode"
```

Comment:
I don't see what is limiting to TGL, this should be "SOF with RT1308 in I2S mode".
There's also no reason why we need the If SND_SOC_SOF_TIGERLAKE.

Reply:
ok. Update the patch once HW test completed.